plugins/search-backend-node/src/runPeriodically.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I borrowed this code from https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/util/runPeriodically.ts
But I don't think that is makes sense to put in into a shared place at it isn't used in the new catalog anymore?